analyses/tp53_nf1_score/util/plot_roc.R

```diff
@@ -8,13 +8,13 @@ suppressPackageStartupMessages({
 })
 
 plot_roc <- function(roc_df, plots_dir, fname){
-
+  
   # add legend 
   roc_df <- roc_df %>%
     mutate(auroc = round(auroc, 2),
            shuffled = ifelse(shuffled, 'TP53 Shuffle', 'TP53'),
            Classifier = paste0(shuffled, ' (AUROC = ', auroc,')'))
-
+  
```

Comment:
These seem like the kind whitespace changes we don't want? Careful with `git add` !